pmmdemo/provision_scripts/postgres_13.yml

```
@@ -22,24 +22,28 @@ runcmd:
   - usermod -a -G docker ec2-user
   - yum install -y https://repo.percona.com/yum/percona-release-latest.noarch.rpm
   - percona-release setup -y ppg-13
+  - percona-release setup -y pmm2-client
```

can we join this one with the one above, i.e. 
```suggestion
  - percona-release setup -y ppg-13 pmm2-client
```

---

Nope, `percona-release` doesn't support it:
```
percona-release setup -y ppg-13 pmm2-client
```
It enables only the first argument.